### ISSUE TITLE
"The One Probe Read Me" is given to players returning from the End

### ISSUE THREAD

**Issue by DuelMonster** [McJty/TheOneProbe#266]
Every time you return from The End via the portal you get given the read me.

**Comment by McJty** [comment]
On what modpack? What version? How are you returning from the end? This is not standard behaviour and certainly does not happen for everyone. That's why I need more info

**Comment by DuelMonster** [comment]
Sorry @McJty, I didn't realize you'd replied...

It is my own combination of mods on a personal server.

If returning from the End from dying, this issue doesn't occur.  It is however reproducible every time you return via the vanilla End portal after having defeated the dragon.

<details><summary>Expand for Mod list:</summary>

```
ActuallyAdditions-1.12.2-r140.jar
BBOutlineReloaded-1.0.1-mc1.12.2.forge.jar
BetterBuildersWands-1.12-0.11.1.245+69d0d70.jar
BetterFps-1.4.8.jar
blockdrops-1.12.2-1.4.0.jar
Chameleon-1.12-4.1.3.jar
ChestTransporter-1.12.2-2.8.8.jar
ChickenChunks-1.12.2-2.4.1.73-universal.jar
ClientTweaks_1.12.2-3.1.8.jar
CodeChickenLib-1.12.2-3.2.1.351-universal.jar
EnderCompass-1.12-1.2.6.2.jar
EnderCore-1.12.2-0.5.37.jar
EnderIO-1.12.2-5.0.31.jar
Enhanced+Selection+Bounding+Box-1.0.1.1.jar
extrautils2-1.12-1.8.5.jar
FlatBedrock-1.4.0_1.12.jar
foamfix-0.10.1-1.12.2.jar
FTBLib-5.3.0.42.jar
FTBUtilities-5.3.0.45.jar
InGameInfoXML-1.12.2-2.8.2.94-universal.jar
InventoryTweaks-1.63.jar
jei_1.12.2-4.11.0.212.jar
journeymap-1.12.2-5.5.2.jar
JustEnoughResources-1.12.2-0.8.9.48.jar
KleeSlabs_1.12.2-5.4.10.jar
LunatriusCore-1.12.2-1.2.0.42-universal.jar
MinersAdvantage-1.12.2-1.3.19.243.jar
moreoverlays-1.14-mc1.12.2.jar
NetherPortalFix_1.12.1-5.3.13.jar
OptiFine_1.12.2_HD_U_E2.jar
QuickLeafDecay-MC1.12.1-1.2.4.jar
refinedstorage-1.6.3.jar
refinedstorageaddons-0.4.1.jar
StorageDrawers-1.12.2-5.3.7.jar
theoneprobe-1.12-1.4.23.jar
```
</details>

**Comment by josephcsible** [comment]
I can confirm this, both with your pack and with just TOP 1.4.23 and Forge 2760, but I note that I don't get the readme again immediately, but rather the next time I save and reload the game.

Update: Also confirmed with just TOP 1.4.25 and Forge 2775, again after saving and reloading.

**Comment by josephcsible** [comment]
@McJty Why do you have this line? It seems to be the problem. https://github.com/McJtyMods/TheOneProbe/blob/e358387cbdc7de108cb37d6bd0660355c79d3159/src/main/java/mcjty/theoneprobe/ForgeEventHandlers.java#L60

**Comment by McJty** [comment]
I have never seen this happen before and this has also never been reported before so I'm wondering what is going on here. Did something change in Forge recently?

Anyway, the reason for that test is to prevent unneeded copy from the player if he didn't die. Because if he didn't die you don't have to clone the capabilities. At least that's how it used to work

**Comment by DuelMonster** [comment]
Probably should have said, but I'm running Forge 2750

**Comment by josephcsible** [comment]
> I have never seen this happen before and this has also never been reported before so I'm wondering what is going on here. Did something change in Forge recently?

Nope, it's worked this way ever since you originally wrote it. Back in 389cd6a, the commit that added it, you were targeting Forge 2105 on 1.10.2: https://github.com/McJtyMods/TheOneProbe/blob/389cd6afc1656d5e7ebc242af218bce493fb756a/build.gradle#L55

I started up a new instance with that and TOP 1.3.0 (the first version that gave out the note), and it happens there too.

**Comment by McJty** [comment]
How are you teleporting from the end to the overworld? I have personally never seen that happen in my lets plays

**Comment by josephcsible** [comment]
Going through the portal that opens after you kill the Ender Dragon. Note that it doesn't immediately give you the manual again when you do that; it happens the first time after you "Save and Quit” then reload (or disconnect then reconnect if on a server) after doing that.

**Comment by kornfan6589** [comment]
I am also having this exact issue. Readme appears in inventory after relog if i have been in the end. running 1.15.2 Forge 31.1.46 and TOP 2.0.2
mod list
https://pastebin.com/vMfwT5We
